Microsoft.Toolkit.Uwp.Notifications/Microsoft.Toolkit.Uwp.Notifications.nuspec

```diff
@@ -0,0 +1,85 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2012/06/nuspec.xsd">
+  <metadata>
+    <id>Microsoft.Toolkit.Uwp.Notifications</id>
+    <version>$version$</version>
+    <title>Windows Community Toolkit Notifications</title>
+    <authors>Microsoft.Toolkit</authors>
+    <owners></owners>
```

Comment:
@michael-hawker If we don't use this, should it be empty or deleted?

Reply:
[Owners is deprecated](https://docs.microsoft.com/en-us/nuget/reference/nuspec#owners) it should be removed, it was replaced with Authors which we have above.

We should ensure our `Authors` contains both `Microsoft.Toolkit,dotnetfoundation`

Reply:
@michael-hawker Alexandre was asking whether we should remove <authors>, <version>, etc... since now I'm using a .nuspec file. Thus the info in the csproj is redundant and not used. I'm in favor of removing the reundant info from the csproj, thoughts?

Reply:
@andrewleader I think removing it from the csproj sounds fine. This comment is in the nuspec though, so I'm a bit confused?

Just validate that the final package has the proper info with the NuGet Package Explorer tool.

Reply:
I'll get this updated today!

Reply:
Added dotnetfoundation as author and removed the deprecated <owners>, and removed the redundant title field from the csproj since that's specified in the nuget. Will re-validate the CI-built package when it's finished to ensure all metadata is correctly there!

Reply:
Changes verified in the CI-built NuGet package!